models/public/hybrid-cs-model-mri/README.md

```diff
@@ -0,0 +1,49 @@
+# hybrid-cs-model-mri
+
+## Use Case and High-Level Description
+
+["A Hybrid Frequency-domain/Image-domain Deep
+Network for Magnetic Resonance Image
+Reconstruction"](https://arxiv.org/pdf/1810.12473.pdf) network. See original [project](https://github.com/rmsouza01/Hybrid-CS-Model-MRI) for the details.
+
+## Specification
+
+| Metric                          | Value                                     |
+|---------------------------------|-------------------------------------------|
+| Type                            | Image inpainting                          |
+| GFlops                          | 146.6037                                  |
+| MParams                         | 11.3313                                   |
+| Source framework                | TensorFlow\*                              |
+
+## Performance
+
+On [validation dataset](https://sites.google.com/view/calgary-campinas-dataset/home) it is reported to achieve `35.772 (mean) +/- 3.214 (std) Db` (PSNR).
```

Comment:
I believe it is accuracy, not performance :)
For reading it is more convenient if you place metrics into table